temporian/beam/__init__.py

```diff
@@ -30,14 +30,19 @@
 """
     )
 
-from temporian.beam import io as _io
-
-read_csv_raw = _io.read_csv_raw
-read_csv = _io.read_csv
-write_csv = _io.write_csv
-to_event_set = _io.to_event_set
-to_dict = _io.to_dict
-UserEventSetFormat = _io.UserEventSetFormat
+from temporian.beam.io import dict as _dict
+from temporian.beam.io import csv as _csv
+from temporian.beam.io import tensorflow as _tensorflow
+
+from_csv_raw = _csv.from_csv_raw
+from_csv = _csv.from_csv
+to_csv = _csv.to_csv
+
+to_event_set = _dict.to_event_set
+to_dict = _dict.to_dict
+
+to_tensorflow_record = _tensorflow.to_tensorflow_record
+from_tensorflow_record = _tensorflow.from_tensorflow_record
```

Comment:
this would be simpler by importing the names directly from each module, since they're not being renamed (`from temporian.beam.io import from_csv_raw, from_csv, to_csv`, etc)